src/panels/config/network/supervisor-hostname.ts

```diff
@@ -110,8 +111,12 @@ export class HassioHostname extends LitElement {
       justify-content: space-between;
       align-items: center;
     }
-    ha-settings-row {
-      border-top: none;
+    .card-content {
+      display: block;
```

Comment:
It will already have display block, as it is a div?

Reply:
Removed. Changed to p